<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileMolGeneratorDialog</class>
 <widget class="QDialog" name="FileMolGeneratorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0,0">
   <item row="4" column="0" colspan="4">
    <widget class="QTextEdit" name="textEdit">
     <property name="lineWrapMode">
      <enum>QTextEdit::NoWrap</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelUnitOfLength">
     <property name="text">
      <string>Unit of Length:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="labelCharge">
     <property name="text">
      <string>Charge:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelGaussiansType">
     <property name="text">
      <string>Gaussians Type:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="comboBoxGaussiansType">
     <item>
      <property name="text">
       <string>Spherical</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Cartesian</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="comboBoxUnitOfLength">
     <item>
      <property name="text">
       <string>Bohr</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Angstrom</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="spinBoxCharge">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimum">
      <number>-99</number>
     </property>
    </widget>
   </item>
   <item row="1" column="2" rowspan="2" colspan="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
